ethereum/client/src/lib.rs

```diff
@@ -158,9 +244,10 @@ impl Contracts {
         }
     }
 
-    pub async fn provide_merkle_root_json(&self, json_string: &str) -> Result<TxHash, Error> {
+    pub async fn provide_merkle_root_json(&self, json_string: &str) -> Result<B256, Error> {
```

Comment:
As I see, it used only in tests, so why not to remove it and use `provide_merkle_root` everywhere instead?

Reply:
Removed, part of code moved to tests.rs